pkg/core/user/role.go

```diff
@@ -0,0 +1,23 @@
+package user
+
+type Role int
+
+const (
+	UserRole Role = iota
```

Comment:
And the word `user` is used here in 2 different contexts: 1. user as an identity for client 2. user as an opposite to admin. I don't know if we should do something with it, maybe NotAdminRole 🤔

Reply:
oh, there is even a topic about it https://ux.stackexchange.com/questions/27769/what-is-a-good-name-for-non-administrator-users :D

Reply:
changed to member